File: javascript/1480. Running Sum of 1d Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    let numbers = [];
    numbers[0] = nums[0];
    for(let i = 1; i < nums.length; i++){
        numbers[i] = nums[i] + numbers[i-1];
    }
    return numbers;
};

File: python/1480. Running Sum of 1d Array.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        result = []
        result.append(nums[0])
        i = 1
        nums.pop(0)
        for num in nums:
            result.append(num + result[i-1])
            i += 1
        return result